ming_sim/action_materialize.py

```
@@ -1082,6 +1082,13 @@ def stage_grant_allocation_candidate(
         n = 0
     if n > 0:
         staged["amount"] = n
+    # #1503：仅显式协饷成案带 purpose=补饷；army 对象的军械/筑城/项目经费不得升格销欠。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)